src/use_case_signup/SignupInputData.java

```diff
@@ -0,0 +1,30 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
+
+package use_case_signup;
+
+public class SignupInputData {
+    private final String username;
+    private final String password;
+    private final String repeatPassword;
```

Comment:
Keep the variable names consistent. Above you call it `password1` and `password2`. I think `repeatPassword` is more informative, so if you could change the name of the password variables to be consistent, that would make readability better.